tests/lib/snaps/test-snapd-autopilot-consumer/wrapper

```diff
@@ -0,0 +1,3 @@
+export GI_TYPELIB_PATH=$SNAP/usr/lib/girepository-1.0:$(ls -d $SNAP/usr/lib/*/girepository-1.0)
+
+$SNAP/usr/bin/python3 $SNAP/provider.py
```

Comment:
AFAIK we can't include `$(...)` calls in snapcraft.yaml's env definitions, pls correct me if I'm wrong. The glob in the path stands for the target architecture, in a local installation of `test-snapd-dbus-provider` it expands to `/snap/test-snapd-dbus-provider/6/usr/lib/x86_64-linux-gnu/girepository-1.0/`